Gruntfile.js

```diff
@@ -181,7 +179,8 @@ module.exports = function(grunt) {
   grunt.registerTask('mocha', ['shell:mochaWithCoverage']);
 
   // Tests
-  grunt.registerTask('test', ['mocha', 'build']); // Test via ts-node. If all goes well, build.
+  grunt.registerTask('lint', ['eslint']);
+  grunt.registerTask('test', ['mocha', 'eslint', 'build']); // Test via ts-node. If all goes well, build.
```

Comment:
Seems suboptimal to me, because `test` task is not running when doing `npm run release` (line #195). Let me know if you want to run `eslint` for `release` as well.